libs/experimental/langchain_experimental/data_anonymizer/utils.py

```diff
@@ -0,0 +1,29 @@
+import string
+from faker import Faker
+
+fake = Faker()
+
+# Listed  entities supported by Microsoft Presidio (for now, global and US only)
+# Source: https://microsoft.github.io/presidio/supported_entities/
+pseudoanonymizer_mapping = {
```

Comment:
constants are usually written in all capital letters with underscores separating words https://peps.python.org/pep-0008/#constants